diff --git a/CMakeLists.txt b/CMakeLists.txt
index a1eca0e..6265e39 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -2,7 +2,7 @@ project (kadu)
 
 # configuration
 
-option (ENABLE_AUTODOWNLOAD "Use automatic download of modules/icons/emoticons" ON)
+option (ENABLE_AUTODOWNLOAD "Use automatic download of modules/icons/emoticons" OFF)
 option (SIG_HANDLING_ENABLED "Define if you want system signals handling" ON)
 option (LIB_SUFFIX_64 "Use suffix 64 in name of library directory" OFF)
 
