diff --git a/kadu-core/gui/windows/about.cpp b/kadu-core/gui/windows/about.cpp
index f824428..99000e1 100644
--- a/kadu-core/gui/windows/about.cpp
+++ b/kadu-core/gui/windows/about.cpp
@@ -154,7 +154,7 @@ About::About(QWidget *parent) :
 	tb_changelog->setFrameStyle(QFrame::NoFrame);
 	tb_changelog->setWordWrapMode(QTextOption::NoWrap);
 	tb_changelog->viewport()->setAutoFillBackground(false);
-	tb_changelog->setText(loadFile("ChangeLog"));
+	tb_changelog->setText(loadFile("about-changes-tab.txt"));
 
 	// add tabs
 	tw_about->addTab(wb_about, tr("&About"));
@@ -213,7 +213,10 @@ QString About::loadFile(const QString &name)
 {
 	kdebugf();
 
-	QFile file(dataPath("kadu/" + name));
+	QString fullName = ("COPYING" == name ?
+			"/usr/share/common-licenses/GPL-2" :
+			"/usr/share/doc/kadu/" + name);
+	QFile file(fullName);
 	if (!file.open(QIODevice::ReadOnly))
 	{
 		kdebugm(KDEBUG_ERROR, "About::loadFile(%s) cannot open file\n", qPrintable(name));
