diff --git a/Plugins.cmake b/Plugins.cmake
index 4bbfca4..835f8ba 100644
--- a/Plugins.cmake
+++ b/Plugins.cmake
@@ -117,11 +117,11 @@ if (UNIX AND NOT APPLE)
 
 	# docking
 		# Indicator docking support
-		indicator_docking
+                #indicator_docking
 
 	# integration
 		# integration with Unity
-		unity_integration
+                #unity_integration
 
 	# notifiers
 		# Freedesktop notification support
