Index: varia/configuration/dialog.ui
===================================================================
--- varia/configuration/dialog.ui	(revision 4894)
+++ varia/configuration/dialog.ui	(working copy)
@@ -40,6 +40,7 @@
 					<item value="Thunderbird" caption="Thunderbird" />
 					<item value="SeaMonkey" caption="SeaMonkey" />
 					<item value="Evolution" caption="Evolution" />
+					<item value="IceDove" caption="IceDove" />
                 </combo-box>
 				<line-edit caption="Custom e-mail client" config-section="Chat" config-item="MailClient" id="mailPath" 
 					   tool-tip="Write specific path to your email client" />
Index: modules/config_wizard/wizard.cpp
===================================================================
--- modules/config_wizard/wizard.cpp	(revision 4894)
+++ modules/config_wizard/wizard.cpp	(working copy)
@@ -488,6 +488,7 @@
 	mailCombo->insertItem(tr("Thunderbird"));
 	mailCombo->insertItem(tr("SeaMonkey"));
 	mailCombo->insertItem(tr("Evolution"));
+	mailCombo->insertItem(tr("IceDove"));
 	connect(mailCombo, SIGNAL(activated(int)), this, SLOT(emailChanged(int)));
 	emailGridLayout->addWidget(mailCombo, 0, 1);
 
Index: kadu-core/main_configuration_window.cpp
===================================================================
--- kadu-core/main_configuration_window.cpp	(revision 4894)
+++ kadu-core/main_configuration_window.cpp	(working copy)
@@ -526,6 +526,12 @@
 			parameters = "mailto:";
 			break;
 		}
+		case 5: // icedove
+		{
+			executableName.append("icedove");
+			parameters = "-compose mailto:";
+			break;
+		}
 	}
 
 	if (emailIndex != 0)
