--- modules/audacious_mediaplayer/spec	2008-03-03 13:08:11.000000000 +0000
+++ modules/audacious_mediaplayer/spec	2008-02-24 20:11:47.000000000 +0000
@@ -1,7 +1,7 @@
 MODULE_SOURCES="audacious.cpp"
 MODULE_MOC_HEADERS="audacious.h"
 MODULE_LDFLAGS="`pkg-config audacious --libs`"
-MODULE_CXXFLAGS="`pkg-config audacious --cflags`"
+MODULE_CXXFLAGS="`pkg-config audacious --cflags` -I/usr/include/dbus-1.0"
 MODULE_CONFIGURE_CMD="pkg-config audacious --exists"
 MODULE_TOOLS="pkg-config which"
 
