diff --git a/kadu-core/icons-manager.cpp b/kadu-core/icons-manager.cpp
index ddd870f..4b0e28c 100644
--- a/kadu-core/icons-manager.cpp
+++ b/kadu-core/icons-manager.cpp
@@ -161,9 +161,17 @@ const QIcon & IconsManager::iconByPath(const QString &path)
 {
 	if (!IconCache.contains(path))
 	{
-		QIcon icon = buildSvgIcon(path);
-		if (icon.isNull())
-			icon = buildPngIcon(path);
+		QIcon icon;
+
+		QFileInfo fileInfo(path);
+		if (fileInfo.isAbsolute() && fileInfo.isReadable())
+			icon.addFile(path);
+		else
+		{
+			icon = buildSvgIcon(path);
+			if (icon.isNull())
+				icon = buildPngIcon(path);
+		}
 
 		IconCache.insert(path, icon);
 	}
