diff --git a/CMakeLists.txt b/CMakeLists.txt
index bae3932..4cb05de 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -56,24 +56,8 @@ if(MINGW)
 	set (CMAKE_SHARED_LIBRARY_PREFIX "")
 endif(MINGW)
 
-if (LIB_SUFFIX_64)
-	set (LIB_SUFFIX 64)
-#else (LIB_SUFFIX_64)
-#	if (UNIX AND NOT APPLE)
-#		if (CMAKE_SIZEOF_VOID_P EQUAL 4)
-#			MESSAGE(STATUS "32 bit architecture detected")
-#			set (LIB_SUFFIX )
-#			set_property (GLOBAL PROPERTY FIND_LIBRARY_USE_LIB64_PATHS FALSE)
-#		endif(CMAKE_SIZEOF_VOID_P EQUAL 4)
-#		if (CMAKE_SIZEOF_VOID_P EQUAL 8)
-#			MESSAGE(STATUS "64 bit architecture detected")
-#			set (LIB_SUFFIX 64)
-#			set_property (GLOBAL PROPERTY FIND_LIBRARY_USE_LIB64_PATHS TRUE)
-#		endif(CMAKE_SIZEOF_VOID_P EQUAL 8)
-#	else (UNIX AND NOT APPLE)
-#		set (LIB_SUFFIX )
-#	endif (UNIX AND NOT APPLE)
-endif (LIB_SUFFIX_64)
+set (LIB_SUFFIX )
+set_property (GLOBAL PROPERTY FIND_LIBRARY_USE_LIB64_PATHS FALSE)
 
 set (DATADIR ${CMAKE_INSTALL_PREFIX}/share)
 set (LIBDIR ${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX})
