diff --git a/modules/mpris_mediaplayer/mpris.cpp b/modules/mpris_mediaplayer/mpris.cpp
index 1b3f785..c5133bb 100644
--- a/modules/mpris_mediaplayer/mpris.cpp
+++ b/modules/mpris_mediaplayer/mpris.cpp
@@ -19,15 +19,6 @@
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
-#include <QtCore/QString>
-#include <QtCore/QStringList>
-#include <QtDBus/QDBusInterface>
-#include <QtDBus/QDBusConnection>
-#include <QtDBus/QDBusMetaType>
-#include <QtDBus/QDBusReply>
-#include <QtCore/QDateTime>
-
-
 #include "modules.h"
 #include "configuration/configuration-file.h"
 #include "misc/path-conversion.h"
@@ -42,7 +33,6 @@ extern "C" int mpris_mediaplayer_init()
 
 	MainConfigurationWindow::registerUiFile(
 		dataPath("kadu/modules/configuration/mpris_mediaplayer.ui"));
-	MainConfigurationWindow::registerUiHandler(mpris);
 
 	bool res = mediaplayer->registerMediaPlayer(mpris, mpris);
 	return res ? 0 : 1;
@@ -52,7 +42,6 @@ extern "C" void mpris_mediaplayer_close()
 {
 	MainConfigurationWindow::unregisterUiFile(
 		dataPath("kadu/modules/configuration/mpris_mediaplayer.ui"));
-	MainConfigurationWindow::unregisterUiHandler(mpris);
 
 	mediaplayer->unregisterMediaPlayer();
 
@@ -73,8 +62,3 @@ void MprisPlayer::configurationUpdated()
 {
 	setService(config_file.readEntry("MediaPlayer", "MPRISService", QString::null));
 }
-
-void MprisPlayer::mainConfigurationWindowCreated(MainConfigurationWindow *mainConfigurationWindow)
-{
-}
-
diff --git a/modules/mpris_mediaplayer/mpris.h b/modules/mpris_mediaplayer/mpris.h
index 3126ee4..b8966f6 100644
--- a/modules/mpris_mediaplayer/mpris.h
+++ b/modules/mpris_mediaplayer/mpris.h
@@ -1,21 +1,39 @@
+/*
+ * %kadu copyright begin%
+ * Copyright 2010 Rafał Malinowski (rafal.przemyslaw.malinowski@gmail.com)
+ * Copyright 2010 Piotr Galiszewski (piotrgaliszewski@gmail.com)
+ * Copyright 2010 Bartłomiej Zimoń (uzi18@o2.pl)
+ * %kadu copyright end%
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 2 of
+ * the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
 #ifndef MPRIS_H
 #define MPRIS_H
 
 #include <QtCore/QString>
 
-#include "gui/windows/main-configuration-window.h"
-
 #include "../mediaplayer/mediaplayer.h"
 #include "../mediaplayer/mpris_mediaplayer.h"
 
-class MprisPlayer : public MPRISMediaPlayer, public ConfigurationUiHandler, ConfigurationAwareObject
+class MprisPlayer : public MPRISMediaPlayer, ConfigurationAwareObject
 {
 	Q_OBJECT
 
 	public:
 		MprisPlayer(QString name, QString service);
 		~MprisPlayer();
-		virtual void mainConfigurationWindowCreated(MainConfigurationWindow *mainConfigurationWindow);
 
 	protected:
 		virtual void configurationUpdated();
@@ -23,4 +41,4 @@ class MprisPlayer : public MPRISMediaPlayer, public ConfigurationUiHandler, Conf
 
 extern MprisPlayer* mpris;
 
-#endif
+#endif // MPRIS_H
